packages/amplify-frontend-ios/tests/amplify-config-helper.test.js

```diff
@@ -1,4 +1,4 @@
-const configHelper = require('../../amplify-frontend-android/lib/amplify-config-helper');
```

Comment:
We have NOT been testing iOS config generator BUT Android all this time from the iOS frontend plugin!